vizro-core/examples/visual-vocabulary/custom_charts.py

```diff
@@ -233,3 +233,70 @@ def dumbbell(data_frame: pd.DataFrame, x: str, y: str, color: str) -> go.Figure:
     # Increase size of dots
     fig.update_traces(marker_size=12)
     return fig
+
+
+@capture("graph")
+def diverging_stacked_bar(
+    data_frame,
+    y: str,
+    category_pos: List[str],
+    category_neg: List[str],
+    color_discrete_map: Optional[Dict[str, str]] = None,
```

Comment:
@antonymilne - could I get your opinion on the API? I've kept it to the minimum, so decided against customising labels etc. Also because it would take me too long to style this properly. 

`data_frame`, `y` and `color_discrete_map` - I've kept similar to the plotly express yntax
`category_pos` and `category_neg` - These are the ones I've introduced. I didn't like any of the existing plotly express argument names as something similar doesn't really exist. `x1` and `x2` don't really cut it for me as a List of columns need to be provided here.

Let me know what you think 🤔

Reply:
tbh I'm not a big fan of `category_pos` and `cateogry_neg` but after playing around with the ordering of legends and traces myself for a couple of hours I absolutely understand why you did it 😅 I have an alternative proposal here - let's discuss this afternoon.